challenges/unit-testing/katas-tdd/calculator/calculator.js

```diff
@@ -0,0 +1,36 @@
+function add(numbers) {
+  let sum = 0;
+  if (typeof numbers !== "string") {
+    throw new Error("Parameter must be a string.");
+  }
+
+  if (numbers === "") {
+    return sum;
+  }
+
+  if (numbers.length === 1) {
```

Comment:
Do yo still need this special case?

Reply:
Yes, I think I still need it because of the way I have written the code. Is there a better way to handle this?

Reply:
I thought both this, and the check on line 16 can be removed together. May be try and check if any test will fail.

Reply:
remove this check will make this test fail ```test("return number if param length is 1", () => {
  const str = "9";
  const currentInput = add(str);
  const targetOutput = 9;
  expect(currentInput).toEqual(targetOutput);
});```

Reply:
I have removed both checks and all test cases passed. Thank you!

Reply:
yep, with TDD usually you will need these kind of conditions to make the tests pass, but once green you can look for simplifications

Reply:
I see! Thank you very much!